packages/apsara-ui/src/FormBuilder/style/horizontal.less

```diff
@@ -0,0 +1,15 @@
+// @import (reference) '../../style/themes/index';
+
+.custom-form-horizontal {
```

Comment:
Is it possible to use `styled-component` for styling ?

Most of the existing components are using `styled-components` and using `less` adds more requirement for maintaining the library.

I was trying to use `sass` as well before but we need to keep it uniform across components 😅 .

Reply:
@utsav14nov We have bee relying on `styled-components`. What's the rationale of using `less` for this?

Reply:
@StewartJingga 
There is no Form component in radix-ui.
I have imported it from rc components from where ant is importing and it uses less and It is tightly coupled with it.
Changing that to a styled-component is overall a different effort and doesn't seems fruitful

Reply:
@utsav14nov you mean rc-component uses less?

Reply:
Yes not rc-field-form, ant uses it, I checked out ant lib of form and build it accordingly. That's why used less here.
and also not much aware that we are sticking to styled-component.

less is being used inside form only and it is independent from all, Can't we take this for now?
We can plan for styled-component in form for future.

Reply:
No, we need to move to styled-components in this PR only. The idea of this refactoring is to actually get rid of all extra dependencies itself.